CHANGELOG.md

```diff
@@ -942,6 +947,26 @@ New modules
 Other minor changes
 -------------------
 
+* Added new proofs to `Algebra.Construct.Add.Identity`
```

Comment:
Oh also no need to add this entry, as it's already covered under the "New modules" header :+1: